config/webpack/webpack.config.js

```diff
@@ -84,14 +89,9 @@ module.exports = (env = {}) => {
     externals: [
       // https://github.com/liady/webpack-node-externals
       externals({
-        allowlist: [
-          'auth0-lock',
-          'js-cookie',
-          'jwt-decode',
-          'loglevel',
-          'util',
-          /babel/,
-        ],
+        modulesFromFile: {
+          excludeFromBundle: ['devDependencies', 'peerDependencies']
```

Comment:
aren't devDeps and peerDeps excluded from prod build already?

Reply:
Good question, not totally sure. However, if I remove `externals`, then the resulting bundle is is ~3.6M whereas it's ~1.7M with the `excludeFromBundle: ['devDependencies', 'peerDependencies']`. To be clear, neither is good ... a bundle this size is crazy ... but it does seem that passing `externals` definitely helps.

<img width="771" alt="Screen Shot 2021-07-09 at 2 55 43 PM" src="https://user-images.githubusercontent.com/440299/125139635-c00a1f00-e0c5-11eb-8923-abddf3cf62f1.png">